HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -5347,10 +5351,11 @@ def self.apply_advanced_defrost(model, htg_coil, air_loop_unitary, conditioned_s
       comp_type_and_control: EPlus::EMSActuatorOtherEquipmentPower
     )
 
+    cnt = model.getOtherEquipments.count { |e| e.endUseSubcategory.start_with? Constants::ObjectTypeBackupSuppHeat } # Ensure unique meter for each heat pump
     defrost_supp_heat_energy_oe = Model.add_other_equipment(
       model,
       name: "#{air_loop_unitary.name} defrost supp heat energy",
-      end_use: Constants::ObjectTypeBackupSuppHeat,
+      end_use: "#{Constants::ObjectTypeBackupSuppHeat}#{cnt + 1}",
```

Comment:
Thanks for the improvements here! This makes me feel we should name it more explicitly if this is the only place to use the `ObjectTypeBackupSuppHeat`, something like `ObjectTypeDefrostBackupSuppHeat`?

Reply:
Good idea. I will make this change.